core/src/main/scala/org/bitcoins/core/api/dlcoracle/OracleEvent.scala

```diff
@@ -60,7 +61,7 @@ sealed trait CompletedOracleEvent extends OracleEvent {
           "Must have a signature for every nonce")
 
   def signatures: Vector[SchnorrDigitalSignature] =
-    nonces
+    nonces.vec
```

Comment:
nit: I would add a `toVector` method on OrderedNocnes rather than using `vec`. But it isn't a big deal.